build.sbt

```diff
@@ -57,8 +57,13 @@ lazy val core = project
         ProblemFilters.exclude[DirectMissingMethodProblem]("ammrunner.Command#Graalvm.launch"),
         ProblemFilters.exclude[DirectMissingMethodProblem]("ammrunner.Command#Jvm.fork")
       )
-    }
+    },
+    buildInfoPackage := "ammrunner",
+    buildInfoKeys := Seq[BuildInfoKey](
+      "scala213" -> scala213
```

Comment:
I'm not sure we should get the Scala 2.13 this way. The actual 2.13 version printed should depend on Ammonite or Scala 3 (which ever scala 2.13 standard library JAR they pull), so maybe we should just hardcode it in the tests.

Reply:
Well it's not really for the tests but rather to know which version of ammonite-cross-23-repl-api to pull down. Because if a user wants to use Scala 3 they need the latest version of `ammonite-cross-23-repl-api` right? Or should we just hard code it there as well?

Reply:
@alexarchambault  ?